examples/drafts/gamma/compare_to_flash_gamma.ipynb

```diff
@@ -0,0 +1,343 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lower_percent_dose_cutoff = 20\n",
+    "local = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pymedphys_gamma_options = {\n",
+    "    'dose_percent_threshold': dose_criteria,\n",
+    "    'distance_mm_threshold': distance_criteria,\n",
+    "    'lower_percent_dose_cutoff': lower_percent_dose_cutoff,\n",
+    "    'interp_fraction': 10,  # Should be 10 or more for more accurate results\n",
+    "    'max_gamma': 1.1,\n",
+    "    'random_subset': None,\n",
+    "    'local_gamma': local,\n",
+    "    'ram_available': 2**29,  # 1/2 GB\n",
+    "    'quiet': False\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pass_rates = flashgamma.gamma_pass_rate(\n",
+    "    ref_dist,\n",
+    "    eval_dist,\n",
+    "    delta_dose=dose_criteria,\n",
+    "    delta_distance=distance_criteria,\n",
+    "    threshold=lower_percent_dose_cutoff,\n",
+    "    local=local\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "np.shape(pass_rates)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# pass_rates"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_passing_rates(passing_rates):\n",
+    "    f = plt.figure()\n",
+    "    ax1 = plt.gca()\n",
+    "\n",
+    "    xx, yy = np.meshgrid(distance_criteria, dose_criteria, )\n",
+    "    im1 = ax1.pcolormesh(xx, yy, passing_rates, shading='nearest')\n",
+    "\n",
+    "    co1 = ax1.contour(distance_criteria, dose_criteria, passing_rates, colors='k', linewidth=1)\n",
+    "    zc = co1.collections\n",
+    "    plt.setp(zc, linewidth=1)\n",
+    "    plt.clabel(co1, fontsize=10, fmt='%1.0f')\n",
+    "\n",
+    "    co2 = ax1.contour(distance_criteria, dose_criteria, passing_rates, colors='#f70c0c', levels=[95])\n",
+    "    zc = co2.collections\n",
+    "    plt.setp(zc, linewidth=2)\n",
+    "    plt.clabel(co2, fontsize=12, fmt='%1.0f')\n",
+    "\n",
+    "    ax1.set_title('Gamma Pass Rates')\n",
+    "    ax1.set_xlabel('Distance to agreement (mm)')\n",
+    "    ax1.set_ylabel('Dose difference (%)')\n",
+    "    c1 = plt.colorbar(im1, ax=ax1)\n",
+    "    c1.set_label('Pass rate (%)', rotation=270, labelpad=12)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FlashGamma's pass rates"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_passing_rates(pass_rates)"
+   ]
+  },
+  {
```

Comment:
might be nice to have additional markdown here to separate the pymedphys calculation from the plotting of the FlashGamma pass rates

Reply:
Thanks @sjswerdloff, I cleaned this up a bit :)